ci/setup-build-test-gdk.sh

```diff
@@ -7,6 +7,27 @@ fi
 
 source /opt/improbable/environment
 
+# Download the unreal-engine.version file from the GDK repo so we can run the example project builds on the same versions the GDK was run against.
+# This is not the pinnacle of engineering, as we rely on GitHub's web interface to download the file, but it seems like GitHub disallows git archive
+# which would be our other option for downloading a single file.
```

Comment:
This part is not relevant for this repo.
Instead I would replace it with saying this is copied from EP's CI for future refactoring purposes.